dist/bundle.js

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
Should the dist folder be on git or is it safe to be removed and put in gitignore?